syntax = "proto3";

package monitor;

service Monitor {
  rpc EventAdded (EventID) returns (MonitorReply);
  rpc TopicRuleAdded (TopicRuleID) returns (MonitorReply);

}

message EventID {
  string id = 1;
}

message TopicRuleID {
  string id = 1;
}

message MonitorReply {
  int32 result = 1;
}